### PR TITLE
Javascript/javascript1/week4

### PR DESCRIPTION
**Pull request by JyothiNandyala**
Adding Javascript1 week4 home work file.

### CODE REVIEW

File: javascript/javascript1/week4/voiceassistent.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+function getReply(command){
+    const commandArray=command.split(' ');
+    
+
+    if(typeof command!=="string"){
+        return "please enter the command in specific formate";
+    }
+    
+    else if(commandArray[0]==="hello"){
```
Comment (ajgcph2019) [inline javascript/javascript1/week4/voiceassistent.js:15]: There are string methods like 'startsWith' and 'endsWith' .Informing for your reference
` const commandArray=command.split(' ');

  if (commandArray.startsWith('hello my name is')) {
    name= commandArray[commandArray.length-1];
    return `Nice to meet you ${name}`;
  }`

File: javascript/javascript1/week4/voiceassistent.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+}
+    console.log("Timer set for "+commandArray[4]+ " " +commandArray[5]);
+    setTimeout(setTimer, milliSeconds);
+    return "Timer is set so just wait";
+}
+if(command){
+    return getMathFunction(commandArray);
+}
+}
+//Mathamatics it will calculate the mathematics operation and return the result.
```
Comment (ajgcph2019) [inline javascript/javascript1/week4/voiceassistent.js:70]: Do you use Prettier for fomatting? There are extra spaces and few issues with alignment of code

File: javascript/javascript1/week4/voiceassistent.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+function getMathFunction(commandArray){
+    let  operation=commandArray[3];
+    let a=parseInt(commandArray[2]);
+    let b=parseInt(commandArray[4]);
+
```
Comment (ajgcph2019) [inline javascript/javascript1/week4/voiceassistent.js:76]: Use meaningful variable names instead of a and b

**Review by ajgcph2019 — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week4/voiceassistent.js (hunk 1 above)
Comment (JyothiNandyala) [reply javascript/javascript1/week4/voiceassistent.js:15]: yes,i made the changes

File: javascript/javascript1/week4/voiceassistent.js (hunk 3 above)
Comment (JyothiNandyala) [reply javascript/javascript1/week4/voiceassistent.js:76]: yes i did it.thank you for remembering.

File: javascript/javascript1/week4/voiceassistent.js (hunk 2 above)
Comment (JyothiNandyala) [reply javascript/javascript1/week4/voiceassistent.js:70]: i have changed the code now.

**Comment by JyothiNandyala** [comment]
Hi Anna,

I did the changes can i merge my pull request,could you please let me know.

thank you.

**Comment by JyothiNandyala** [comment]
made the changes based on mentors feedback.

**Pull request merged**